pkg/binary/envoy/controlplane/istio_test.go

```diff
@@ -38,43 +28,3 @@ func TestMain(m *testing.M) {
 	}
 	os.Exit(m.Run())
 }
-
-// NOTE: This test will fail on macOS due to an issue with Envoy, the same issue as debug logging
-func Test_IstioGateway(t *testing.T) {
```

Comment:
most of the code here doesn't exist anymore. Not sure if we want to do this or delete the test. Seems possibly easier in docker?

Reply:
@liamawhite Could you please advise how to upgrade correctly Istio dependencies without breaking `getenvoy run` ?

I think, this unit test is only one aspect to consider.

More importantly, we need to make sure that
```shell
# Run as a gateway using an Istio controlplane bootstrap.
getenvoy run standard:1.11.1 --mode loadbalancer --bootstrap istio --controlplaneAddress istio-pilot.istio-system:15010
```
will not get broken.